frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.module.css

```diff
@@ -10,8 +10,16 @@ body {
 
 .EmbedFrame {
   color: var(--mb-color-text-primary);
-  background-color: var(--mb-color-bg-dashboard);
 }
+
+.EmbedFrame,
+/* this is to make it work when exporting to pdfs, where the EmbedFrame is not part of the exported dom */
+.WithThemeBackground {
+  /* the dashboard background color is not set by default in the embedding sdk */
+  background-color: var(--mb-color-bg-dashboard, --mb-color-background);
```

Comment:
This doesn't work. The actual syntax should be

```ts
  background-color: var(--mb-color-bg-dashboard, var(--mb-color-background));
```

See the `var` on the fallback value.

![image](https://github.com/user-attachments/assets/314935da-37ee-4ede-a3b1-0bf62860869d)

If you already tested that this code works as expected that means you can just drop the second variable.

```suggestion
  background-color: var(--mb-color-bg-dashboard);
```

Reply:
@WiNloSt Great catch! I just realized we already fallback dashboard background color to the background color. Fixed in https://github.com/metabase/metabase/pull/48645/commits/836c28b7c144e188e76a8df4b92594f37564899c - tested again and it works.